### ISSUE TITLE
Check physics

### ISSUE THREAD

**Issue by pb-uk** [lightcone7/light-cone-calc#6]
Some of the calculations in the legacy code look odd - particularly radiation which may be using s + 1 and s + 2 instead of s and s + 1 (or z + 1 and z + 2).

**Comment by burtjordaan** [comment]
Yes, There is an error in the Legacy js code, line 356 should read
 var OmegaM =  (Omega-Ynow * Ynow / Yinf / Yinf) * s_eq / (1+s_eq);
Tiny, tiny influence, considering the uncertainty on the z_eq value.
